docs/resources/aws_ebs_snapshot.md

```diff
@@ -0,0 +1,116 @@
+---
+title: About the aws_ebs_snapshot Resource
+platform: aws
+---
+
+# aws\_ebs\_snapshot
+
+Use the `aws_ebs_snapshot` InSpec audit resource to test properties of a single AWS EBS snapshot. These are point-in-time
+incremental backups of AWS EBS volumes that are saved to AWS S3.
+
+## Syntax
+
+Ensure an EBS snapshot exists:
+
+    describe aws_ebs_snapshot(snapshot_id: 'SNAPSHOT_ID') do
+      it { should exist }
+    end
+
+You may also use hash syntax to pass the EBS volume name:
+
+    describe aws_ebs_snapshot(name: 'SNAPSHOT_ID') do
+      it { should exist }
+    end
+
+### Parameters
+
+This resource accepts a single parameter, either the EBS snapshot ID or name (from the Name tag). At least one must be provided.
+
+#### snapshot\_id
+
+The EBS snapshot ID which uniquely identifies the volume.
+This can be passed as either a string or an `snapshot_id: 'value'` key-value entry in a hash.
+
+For additional information, see the [AWS documentation on EBS snapshots](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSSnapshots.html).
+
+#### name _(required if `snapshot_id` not provided)_
+
+The EBS volume name from the name tag. This must be passed as a `name: 'value'` key-value entry in a hash.
+It is not advised to use this parameter if your Name tags for your snapshots are not unique, as at most one entry is returned.
+
+## Properties
+
+|Property     | Description|
+| ---         | --- |
+|snapshot\_id | The unique ID for the EBS snapshot. |
+|tags         | A hash of tags for the EBS snapshot, e.g. {'Name' => 'snapshot-name'} . |
+|encrypted    | A boolean indicating whether the snapshot is encrypted. |
+|group        | Either set to `all` if the EBS snapshot is public (anyone can create a volume from the EBS snapshot), or `nil`. |
+|user\_ids    | An array of user\_ids (account numbers) that have been granted create volume permission, or an empty array if no other accounts have been granted permission to create a volume from this EBS snapshot. |
+
+There are also additional properties available, for instance `description`, `owner_id` and `volume_size`. For a comprehensive list,
```

Comment:
Can the full list of available properties get added to this.

Reply:
Done. I also did a s/EBS snapshot/EBS Snapshot/g in both docs.